test/membrane_element_rtp_aac/pipeline_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule Membrane.RTP.AAC.Pipeline.Test do
+  use ExUnit.Case, async: true
+
+  import Membrane.ChildrenSpec
+  import Membrane.Testing.Assertions
+  alias Membrane.RTP.AAC.Utils
+
+  describe "test payloader to depayloader is identity" do
+    @spec run_pipeline_test(pos_integer(), Utils.mode(), binary()) :: pid()
+    defp create_pipeline_supervised!(frames_per_packet, mode, payload) do
+      spec = [
+        child(:source, %Membrane.Testing.Source{
+          output: payload,
+          stream_format: %Membrane.AAC{}
+        })
+        |> child(:payloader, %Membrane.RTP.AAC.Payloader{
+          mode: mode,
+          frames_per_packet: frames_per_packet
+        })
+        |> child(:depayloader, %Membrane.RTP.AAC.Depayloader{mode: mode})
+        |> child(:sink, Membrane.Testing.Sink)
+      ]
+
+      Membrane.Testing.Pipeline.start_link_supervised!(spec: spec)
+    end
+
+    defp run_pipeline_test(frames_per_packet, mode, payload) do
+      pipeline_pid = create_pipeline_supervised!(frames_per_packet, mode, payload)
+      assert_sink_playing(pipeline_pid, :sink)
+
+      assert_sink_buffer(pipeline_pid, :sink, %Membrane.Buffer{
+        payload: ^payload
+      })
+
+      assert_end_of_stream(pipeline_pid, :sink)
```

Comment:
It would be helpful to typespec through Membrane.Testing

Reply:
Could you elaborate on that? :D

Reply:
nvm, looks like dialyzer complains when helpers are public and without a typespec, but making them private resolves the issue